server/model/adjustmentTypes.ts

```diff
@@ -54,12 +54,6 @@ const adjustmentTypes: AdjustmentType[] = [
     shortText: 'special remission',
     url: 'special-remission',
   } as AdjustmentType,
-  {
```

Comment:
Since TIME_SPENT_IN_CUSTODY_ABROAD was removed from the API it was causing an eslint error here, so removed here as well